README.md

```diff
@@ -1,2 +1,41 @@
 # raygun4net-examples
-A variety of sample apps which are compatible with raygun4net
+
+A variety of example apps which are compatible with raygun4net
+
+## Setup
```

Comment:
How do you run the projects? I'm guessing to run both Android and iOS apps is to simply run `dotnet run` ?

Reply:
Correct! running instructions for each individual project are in the READMEs of the project, this is just for general setup of the solution, like in `Raygun4Net.NetCore.Example.Android/README.md`